index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   <div id="header">
+     <img id="header-img" src="https://helpx.adobe.com/content/dam/help/en/photoshop/using/convert-color-image-black-white/jcr_content/main-pars/before_and_after/image-before/Landscape-Color.jpg" alt="">
+     <nav id="nav-bar">
+         <a class="nav-link" href="#form">Subscribe</a>
+         <a class="nav-link" href="#video">How It Works</a>
+         <a class="nav-link" href="#pricing">Price</a>
+     </nav>
+   </div>
```

Comment:
I think it would be nicer if you use the 'header' tag instead of a regular 'div'.